glide.lock

```diff
@@ -1,10 +1,12 @@
-hash: f3e767e8346d32c0a86b9c441a9f5a580a4da1e4e9b201cffc7e2b313149bc5d
-updated: 2017-11-14T15:40:02.556527845+05:30
+hash: 9be3af04ee1417f46551109c7ed120aa3bb93a56a471144a7be35464d180f775
```

Comment:
The vendor changes, glide changes should go in one commit and the rest of the code changes should go in another commit!